test/Microsoft.AspNetCore.Mvc.IntegrationTests/ParsingSimpleTypeModelBinderIntegrationTest.cs

```diff
@@ -0,0 +1,507 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.Extensions.Primitives;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.IntegrationTests
+{
+    public class ParsingSimpleTypeModelBinderIntegrationTest
```

Comment:
Do we really need 500 lines of tests for this? I thought the unit tests were pretty exhaustive. IMO it would be good enough to show an integration test that by default if you make a model with one of the supported types we will support thousands.